hack/bootstrap-ginkgo.sh

```diff
@@ -4,12 +4,16 @@ set -e
 
 source $(dirname "$0")/common.sh
 
-_out/tests/ginkgo version
+FOLDERS="${KUBEVIRT_DIR}/staging/src/kubevirt.io/ ${KUBEVIRT_DIR}/pkg/ ${KUBEVIRT_DIR}/cmd/ ${KUBEVIRT_DIR}/tests/framework/"
+
+ginkgobin=$(realpath _out/tests/ginkgo)
+$ginkgobin version
```

Comment:
Do we just print it here for debugging purposes?

Reply:
Yeah though it's not needed anymore since we don't `|| true` the ginkgo bootstrap call
Will remove